src/assert.js

```diff
@@ -77,6 +77,18 @@ QUnit.assert = Assert.prototype = {
 	},
 
 	/**
+	 * Asserts rough false-ish result.
+	 * @name notOk
+	 * @function
+	 * @example notOk( "asdfasdf".length <= 5, "There must be at max 5 chars" );
```

Comment:
This seems confusing. This assertion is not a good example.

Although, I filed the PR #746 to remove these JSDocs, let's see if we can get rid of them.